about.html

```diff
@@ -8,8 +8,8 @@
 <h1>About</h1>
 
 <p>
-  About us: We're just a couple of Square-1 enthusiasts who wanted to learn how to code
-  our own website.
+  About us: We're a couple of Square-1 enthusiasts who wanted to show others the elegance and efficiency of OBL
+  with our expertise in creating websites.
```

Comment:
hm, maybe this is too extra lol

Reply:
yeah, kind of too extra especially since you have no *expertise* in creating websites :D